roles/percona-server/tasks/main.yml

```diff
@@ -46,6 +46,29 @@
   until: result|succeeded
   retries: 5
 
+# Upgrade to the latest xtradb for CVEs
+# This needs to be done serially because it'll restart the damn thing on
+# package upgrade.
+- block:
+  - name: upgrade percona to desired version
+    apt:
+      pkg: percona-xtradb-cluster-server-{{ xtradb.server_version }}
+      state: latest
```

Comment:
This is disingenuous as it's specifying the latest version whereas the name states a "desired version"
Where is `{{ xtradb.server_version }}` defined in role defaults?